### PR TITLE
Set-up-contact-form

### PR DESCRIPTION
**Pull request by Donard97**
Today I and my partner Lilian did a section of the contact us form. We were requested to make sure that we guide our users' experience in a way that helps them fill in the form without frustration.

# General requirements
1.Make sure that there are no linter errors.
2.Make sure that you used the correct GitHub flow.
3.Make sure that you documented your work in a professional way.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -224,5 +224,71 @@ <h3>Skills</h3>
...
     </div>
+
+    <section class="contact-form">
+      <div class="contact-header">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+      </div>
+      <form
```
Comment (vichuge) [inline index.html:235]: - [ ] Please, add a `action` tag with the [formspree link](https://formspree.io/html). Remember that you need to use it on your form according to the [Requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Collect%20data%20by%20using%20the%20Formspree%20service.) 🐶

File: index.html [hunk 2]
```diff
@@ -224,5 +224,71 @@ <h3>Skills</h3>
...
+        id="form"
+        name="simple-contact"
+        accept-charset="utf-8"
+        method="post"
```
Comment (vichuge) [inline index.html]: - [ ] The method could be on uppercase. `method="POST"` 🐱

File: index.html [hunk 3]
```diff
@@ -224,5 +224,71 @@ <h3>Skills</h3>
...
+      >
+        <fieldset id="fieldset-inputs">
+          <input
+            type="text"
+            name="name"
+            id="full-name"
+            placeholder="Full name"
+            required=""
+          />
```
Comment (vichuge) [inline index.html]: - [ ] Name input should have maxlength="30" attribute. Please work on it. 🌵
- [ ] For require attribute, try to use `required` in place of `required=""`. This specific attribute doesn't use the value assignment.

File: index.html [hunk 4]
```diff
@@ -224,5 +224,71 @@ <h3>Skills</h3>
...
+            placeholder="Email address"
+            required=""
+          />
+          <textarea
+            rows="5"
+            name="message"
+            id="message"
+            placeholder="Enter text here"
+            required=""
+          ></textarea>
```
Comment (vichuge) [inline index.html]: - [ ] Message input has `maxlength="500"` attribute, please work on it. 🦊
- [ ] For require attribute, try to use `required` in place of `required=""`. This specific attribute doesn't use the value assignment.

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, consider:
- [OPTIONAL]

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @Donard97 and @ifnotlily , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**